packages/bsb/docs/config/types.rst

```diff
@@ -138,8 +139,9 @@ configured string will be fetched from the referenced node:
     "where": "A"
   }
 
-Assuming that ``where`` is a reference to ``locations``, location ``A`` will be retrieved
-and placed under ``where`` so that in the config object:
+Assuming here that ``where`` is a reference attribute, referring to ``locations``,
+Location ``A`` will be retrieved during the `Configuration` building and placed under
+``where``. Once this is done, you can access the configuration attributes normally:
```

Comment:
Again I think it's confusing to explain that it's not ready during config building, while it's impossible for users to deal with incompletely built configurations.